spec/policies/idv/flow_policy_spec.rb

```diff
@@ -159,7 +161,8 @@
 
     context 'preconditions for agreement are present' do
       it 'returns agreement' do
-        idv_session.welcome_visited = true
+        stub_up_to_key(key: :welcome, idv_session: idv_session)
```

Comment:
I think this would read a little smoother:

```suggestion
        stub_up_to(:welcome, idv_session: idv_session)
```

Reply:
Good idea! Done in e54aee08759cc689ce46351eb26ce3dad0f02233